src/com/facebook/buck/testrunner/TestNGRunner.java

```diff
@@ -134,6 +141,79 @@ public void transform(ITestAnnotation annotation, Class testClass, Constructor t
     }
   }
 
+  /**
+   * Listens for suite configuration / failure, captures stdout/stderr when configuring for
```

Comment:
Not sure what you mean - that comment is basically saying 'suite configuration or failure'